apps/laboratory/src/types/EIP5792.ts

```diff
@@ -0,0 +1,8 @@
+export type CapabilityName = 'atomicBatch' | 'paymasterService' | 'sessionKey'
+export type Capabilities = {
+  [K in CapabilityName]?: {
+    supported: boolean
+  }
+}
+export type GetCapabilitiesResult = Record<string, Capabilities>
```

Comment:
what would be the key to this record? an address?

Reply:
chainId in hex